app/services/document_capture_session_result.rb

```diff
@@ -28,7 +28,7 @@ def selfie_status
 
   def success_status
     # doc_auth_success : including document, attention_with_barcode and id type verification
-    !!doc_auth_success && selfie_status != :fail
+    !!doc_auth_success && selfie_status != :fail && !!pii
```

Comment:
This does check pii completely not available but not partially pii failure?

Reply:
pii doesn't exist if it's not valid